src/styles/components/info-list.scss

```diff
@@ -24,6 +28,23 @@
   .info-list {
     &__item {
       display: block;
+      &__title {
+        text-align: left;
+      }
+      &__content {
+        border-left: none;
+        padding-left: 0;
+      }
+    }
+  }
+}
+
+@include breakpoint(medium) {
+  .info-list--columns {
+    column-count: 2;
+    column-width: 50%;
+    .info-list__item {
```

Comment:
Is `item` a modifier of `columns` or just a different element?

Reply:
`columns` is the modifier of the top level element, `item` doesn't change